README.md

```diff
@@ -13,55 +13,61 @@
 
 For further information, check [VATZ Project Design](docs/design.md)
 
 
 ## What is the key feature of **VATZ Project**?
 
-  ### Multi Protocol Support
-   **VATZ** is NOT limited Protocol Type where it categorizes on chain protocol. Any Protocol can be managed through VATZ with plugins, even unsupported protocols can be integrated through simple plugin development.
-  ### Infrastructure as Code
-   **VATZ** is described using a high-level configuration syntax. You can divide your plugins into modular components that can then be combined in different ways to behave through automation.
-  ### Data Analysis
-   **VATZ** helps to build datasets for your managing protocols and transfer your data into popular services Prometheus, Kafka, Google BigQuery and more. Because of this, VATZ aims to set your Node infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure. (2023-Q3)
-  ### Change automation
-  Complex sets of node's operational tasks can be done through **VATZ** with minimal human interaction. (2023-Q4)
+### Multi-Protocol Support
+Any protocol can be managed through **VATZ** with plugins. Even unsupported protocols can be integrated through simple plugin development.
 
+### Infrastructure as Code
+**VATZ** is described using a high-level configuration syntax. You can divide your plugins into modular components that can then be combined in different ways to behave through automation. 
 
-# Our Mission
+### Monitoring
+Various logs and node data are collected and exported by a node exporter, then monitored through the 3rd party applications like Grafana. 
+
+### Data Analysis
+**VATZ** helps build datasets for your protocols, and transfer your data into popular services like Prometheus, Kafka, Google BigQuery, and more. In this way, **VATZ** aims to optimize your node infrastructure, and operators get insight into dependencies in their infrastructure. (Exp. 2023-Q3)
 
-We're on a mission to transform the way people experience blockchain technology and let them contribute and become a part of its technology.
-As Validators, we provide tools to people to manage their own nodes with low cost and less effort for anyone who would like to join future blockchain technology.
+### Change Automation
+Complex node operation tasks can be executed through **VATZ** with minimal human interaction. (Exp. 2023-Q4)
+
+# Our Mission
+We're on a mission to both transform the way people experience blockchain technology, and help them shape it. 
+As validators, we provide tools for low-cost and low-effort node management for anyone wanting to onboard next-generation blockchain technology.
 
 --- 
 # Usage of VATZ
 
-## How to get started with **VATZ**?
-Please follow [Installation guide](docs/installation.md) to install and start VATZ.
+## How to get started with VATZ
+Check out the[Installation Guide](docs/installation.md) to install and start using VATZ.
```

Comment:
How about modifying the documentation link like this?
`docs/installation.md` -> `./docs/installation.md`
<img width="542" alt="image" src="https://github.com/dsrvlabs/vatz/assets/19239345/4b2a4d83-e159-46b6-a4e1-cdc4dd0d1426">